src/components/Dashboard/TodoList.vue

```diff
@@ -58,7 +58,7 @@
 
                 <!-- blue details button -->
                 <v-btn
-                  v-else-if="showDetailsBtnBlue(item)"
+                  v-else-if="showDetailsBtnBlue(item) || isAffiliationInvitation(item)"
```

Comment:
Instead of parallel conditions throughout this (already-ugly) template, could the affiliation condition be added to `showDetailsBtnBlue`?